cub/cub/device/dispatch/tuning/tuning_scan_by_key.cuh

```diff
@@ -63,12 +63,6 @@ enum class primitive_op
   no,
   yes
 };
-enum class offset_size
-{
-  _4,
-  _8,
-  unknown
-};
```

Comment:
Is that intentional?

Reply:
I think so, the enum is not used in this file. Probably a copy&paste error from the past.